packages/core/EventManager.ts

```diff
@@ -266,6 +266,10 @@ export default class EventManager {
     };
   }
 
+  public async addAttendee(event: CalendarEvent, booking: PartialBooking) {
+    await this.updateAllCalendarEvents(event, booking);
+  }
```

Comment:
Create a new method that only calls on updating the calendar, without the added logic of rescheduling